pkg/scheduler/framework/plugins/nodevolumelimits/utils.go

```diff
@@ -29,6 +31,38 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/framework"
 )
 
+// NOTE: constants in this file are copied from pkg/volume/util package to avoid external scheduler dependency
```

Comment:
An important part of this is to avoid code duplication, so we don't just want to copy constants/functions we want to actually move them and update all callers to refer to the new location.

So the code in this PR may be a better fit to move to component-helpers. Updating the references in other places will also force us to get review from other sigs, which is good (I think sig-storage would be most relevant to this)

Reply:
Yes, indeed I want to go further here. After some investigation, I think it is time for us to entirely remove some dependencies here. Ref #96852.